src/index.html

```diff
@@ -165,5 +180,60 @@ <h2 class="modal__title">New Post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <div class="modal__header">
+          <h2 class="modal__title">Edit profile avatar</h2>
+
+          <button
+            type="button"
+            id="modal-close-btn"
+            class="modal__close-btn"
+          ></button>
+        </div>
+        <form name="edit-avatar-form" class="modal__form" novalidate>
+          <label class="modal__label" for="profile-avatar-input">
+            Profile picture
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/xe8Ka6.jpg